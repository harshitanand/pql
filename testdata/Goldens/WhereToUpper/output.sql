SELECT * FROM "StormEvents" WHERE coalesce((UPPER("EventType")) = 'TORNADO', FALSE);
