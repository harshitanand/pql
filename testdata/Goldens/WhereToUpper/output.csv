EventId,State,EventType,DamageProperty
60913,FLORIDA,Tornado,6200000
